packages/shared/lib/core/market/utils/getMarketAmountFromAssetValue.ts

```diff
@@ -0,0 +1,7 @@
+import { IAsset } from '@core/wallet'
+import { getMarketPriceForAsset } from './getMarketPriceForAsset'
+
+export function getMarketAmountFromAssetValue(amount: number, asset: IAsset): number {
+    const marketPrice = getMarketPriceForAsset(asset)
+    return (marketPrice * amount) / 10 ** asset?.metadata?.decimals
```

Comment:
maybe we need some check for the amount of `asset?.metadata?.decimals` because if they are too large, it could crash the app